src/Cake.FileHelpers/Cake.FileHelpers.csproj

```diff
@@ -19,7 +19,7 @@
     <Copyright>Copyright 2017-$([System.DateTime]::Now.Year) - Cake Contributions</Copyright>
     <PackageProjectUrl>https://github.com/cake-contrib/Cake.FileHelpers</PackageProjectUrl>
     <PackageIcon>icon.png</PackageIcon>
```

Comment:
You can remove this line, as well as the the `icon.png` file. `CakeContrib.Guidelines` adds those automatically, now.

Reply:
Awesome. Removed!